src/briefcase/bootstraps/toga.py

```diff
@@ -71,6 +71,10 @@ def pyproject_table_linux_system_debian(self):
     "libcairo2-dev",
     # Needed to compile PyGObject wheel
     "libgirepository1.0-dev",
+    # Needed to run the app in Docker
```

Comment:
Why "in docker"? Won't this be a requirement regardless - just one that virtually every Debian install will have by default?

Reply:
> Won't this be a requirement regardless

It's only a requirement to _run_ the app....not to build it. If you aren't using Docker, then Briefcase will tell you the `system_requires_runtimes` list needs to be installed first.

But I guess this brings up a larger point....

By adding these runtime requirements to `system_requires`, we're really co-opting its purpose which to specify the build-time requirements.

Maybe we should just have the `Dockerfile` install both...

Reply:
Instead of duplicating the runtime requirements in to `system_requires`, I just updated the Dockerfile invocation to install both `system_requires` and `system_runtime_requires`. This makes so much more sense in retrospect.